x-pack/plugins/observability_solution/infra/public/components/logging/log_minimap/log_minimap.tsx

```diff
@@ -157,23 +157,23 @@ export class LogMinimap extends React.Component<LogMinimapProps, LogMinimapState
   }
 }
 
-const MinimapBorder = euiStyled.line`
-  stroke: ${(props) => props.theme.eui.euiColorMediumShade};
+const MinimapBorder = styled.line`
+  stroke: ${(props) => props.theme.euiTheme.colors.mediumShade};
   stroke-width: 1px;
 `;
 
-const TimeCursor = euiStyled.line`
+const TimeCursor = styled.line`
   pointer-events: none;
   stroke-width: 1px;
   stroke: ${(props) =>
-    props.theme.darkMode
-      ? props.theme.eui.euiColorDarkestShade
-      : props.theme.eui.euiColorDarkShade};
+    props.theme.colorMode === 'DARK'
+      ? props.theme.euiTheme.colors.darkestShade
```

Comment:
**nit**: is this colour deprecated and should it change with another one from the palette?